.circleci/config.yml

```diff
@@ -29,7 +29,8 @@ jobs:
     - run: git config --global user.email "circle@circleci.com"
     - run: git config --global user.name "CircleCI"
     - run: chmod -Rv +x scripts/
-    - run: ./scripts/build.sh 
+    #- run: ./scripts/build.sh 
+    - run: git submodule update --init --recursive
```

Comment:
Something with this seems wrong, but I am not sure what it is.
I am also trying to push the usage of Jenkins instead of CircleCI or Travis, the builds do work on Jenkins.
HTTP://josephworks.ddns.net:8080/job/Akarin-project/